documentation/src/main/asciidoc/userguide/chapters/query/hql/HQL.adoc

```diff
@@ -1780,6 +1791,20 @@ include::{sourcedir}/HQLTest.java[tags=hql-like-predicate-escape-example]
 ----
 ====
 
+[[hql-ilike-predicate]]
+=== Ilike predicate
+
+Performs a case-insensitive like comparison on string values. The syntax is:
+
+[[hql-ilike-predicate-bnf]]
+[source, JAVA, indent=0]
+----
+include::{extrasdir}/predicate_ilike_bnf.txt[]
+----
+
+The semantics are identical to those of the aforementioned <<hql-like-predicate>>, with the sole difference that the comparison is now case insensitive. Note that for this to work the database will have to be set to a case sensitive collation.
```

Comment:
>Note that for this to work the database will have to be set to a case sensitive collation.

Well, that is not really true I think. The case insensitive comparison works regardless of the collation. The case sensitive like is sensible to the collation though.

Reply:
Yeah, you're right, i'll scratch this.